gc/base/OMRVMInterface.cpp

```diff
@@ -81,6 +81,15 @@ GC_OMRVMInterface::getOmrHookInterface(MM_GCExtensionsBase *extensions)
 }
 
 /**
+ * Return the OMR private hook interface for the Memory Manager
+ */
+J9HookInterface**
+GC_OMRVMInterface::getOmrPrivateHookInterface(MM_GCExtensionsBase *extensions)
+{
```

Comment:
Can you expose the `getOmrPrivateHookInterface` on the J9 side without making access to it part of the api?